hack/create.sh

```diff
@@ -15,7 +15,8 @@ set -x
 USE_PROD_FLAG="-use-prod=false"
 if [[ -n "$TEST_IN_PRODUCTION" ]]; then
     USE_PROD_FLAG="-use-prod=true"
+else
+	hack/fakerp.sh $RESOURCEGROUP &
 fi
 
-go generate ./...
```

Comment:
Any particular reason this got removed?

Reply:
Runs as part of `fakerp.sh` now.

Reply:
I would like this better in main 'root' script at the top somewhere?

Reply:
What do you mean by root script? hack/create.sh? Generation is related to the backend code run by the fake RP so it's natural place is fakerp.sh.